torch/_subclasses/fake_tensor.py

```diff
@@ -334,14 +334,6 @@ def to_copy(fake_mode, func, *args, **kwargs):
         return FakeTensor(fake_mode, aten._to_copy(input, **new_kwargs), out_device)
 
 
-@register_op_impl(aten.clone.default)
```

Comment:
how is this dead, is it overshadowed by another implementation?  Don't we error on duplicates?

Reply:
The C++ impl natively supports symint, and no, no error in this case